src/aten_detector.erl

```diff
@@ -95,8 +106,7 @@ notify(_Watchers, [], _Evt) ->
 notify(Watchers, [Node | Nodes], Evt) ->
     case Watchers of
         #{Node := Pids} ->
-            [Pid ! {node_event, Node, Evt}
-             || Pid <- Pids];
+            maps:map(fun(Pid, _) -> Pid ! {node_event, Node, Evt} end, Pids);
```

Comment:
When throwing away return values I prefer to use an explicit `_ = maps:map...` to highlight the fact you intend to discard the return value and return `ok`.